WalletWasabi/Legal/Assets/LegalDocuments.txt

```diff
@@ -155,16 +161,16 @@ I. TERMS AND CONDITIONS
     You and the Service Provider agree to arbitrate any dispute arising under or in connection with these Terms or Your use of our Services, except for disputes in which either party seeks equitable and other relief for any alleged infringement or unlawful use of copyrights, trademarks, trade names, logos, trade secrets, patents or other intellectual property rights. Arbitration prevents You from suing in court or from having a jury trial. You and the Service Provider agree to notify each other in writing of any dispute within thirty (30) days of when it arises. Notice to the Service Provider shall be sent to legal@zksnacks.com.
 
     You and the Service Provider further agree:
-     - to attempt informal resolution prior to any demand for arbitration;
-     - that the seat or legal place of any arbitration will be Gibraltar;
-     - that arbitration will be conducted confidentially by a single arbitrator appointed by the Gibraltar Chamber of Commerce;
+- to attempt informal resolution prior to any demand for arbitration;
+     - that the seat or legal place of any arbitration will be Republic of Seychelles;
+     - that arbitration will be conducted confidentially by a single arbitrator appointed by law of Republic of Seychelles;
      - the arbitration shall be conducted in accordance with the rules of International Chamber of Commerce; and
-     - that the courts in Gibraltar will have exclusive jurisdiction over any appeals of an arbitration award.
-    Other than class procedures and remedies discussed below, the arbitrator has the authority to grant any remedy that would otherwise be available in court.
-    Whether the dispute is heard in arbitration or in court, You and the Service Provider will not commence against the other any class action, class arbitration or representative action or proceeding.
+     - that the arbitration in Republic of Seychelles will have exclusive jurisdiction over any appeals of an arbitration award.
+   Other than class procedures and remedies discussed below, the arbitrator has the authority to grant any remedy that would otherwise be available in court.
```

Comment:
```suggestion
    Other than class procedures and remedies discussed below, the arbitrator has the authority to grant any remedy that would otherwise be available in court.
```

Reply:
Why?